Dockerfile

```diff
@@ -27,14 +27,49 @@ COPY --from=builder /home/jpo-conflictmonitor/src/main/resources/application.yam
 COPY --from=builder /home/jpo-conflictmonitor/src/main/resources/logback.xml /home
 COPY --from=builder /home/jpo-conflictmonitor/target/jpo-conflictmonitor.jar /home
 
+
+
+# Use jemalloc for RocksDB per Confluent recommendation:
+# https://docs.confluent.io/platform/current/streams/developer-guide/memory-mgmt.html#rocksdb
+RUN amazon-linux-extras install -y epel
+RUN yum install -y jemalloc-devel
```

Comment:
Might be best to combine these in one line and remove the downloaded apt lists, [relevant docker best practices](https://docs.docker.com/build/building/best-practices/#sort-multi-line-arguments)

Reply:
Ok, combined the commands, but amazon linux uses yum, not apt, so can't delete apt lists (and I can't find any info about if yum has some equivalent that should be removed)